sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/Utility.java

```diff
@@ -482,4 +488,25 @@ public static SharedKeyCredential getSharedKeyCredential(HttpPipeline httpPipeli
         }
         return null;
     }
+
+    public static Flux<ByteBuffer> convertStreamToByteBuffer (InputStream data, long length, int blockSize ) {
```

Comment:
Mind fixing the space before the parentheses?

Reply:
The method can be ignored in this PR. Will merge in the correct one from playback client one.